site2/docs/functions-develop.md

```diff
@@ -656,6 +656,182 @@ $ bin/pulsar-admin functions create \
 
 All logs produced by `LoggingFunction` above can be accessed via the `persistent://public/default/logging-function-logs` topic.
 
+#### Customizing Function Log Level
+Additionally, you can use the XML file, `functions_log4j2.xml`, to customize the function log level. 
+To customize the function log level, create or update `functions_log4j2.xml` in your Pulsar conf directory (e.g. `/etc/pulsar/`) to contain contents such as:
```

Comment:
```suggestion
To customize the function log level, create or update `functions_log4j2.xml` in your Pulsar conf directory (for example, `/etc/pulsar/`) to contain contents such as:
```

Reply:
"e.g.," actually means the same thing as "for example," 
Would you still like me to make this change?

Reply:
I noticed your other comment about not using contractions. I assume it's the same point here.